biggus/__init__.py

```diff
@@ -2597,6 +2787,103 @@ def _sliced_shape(shape, keys):
     return sliced_shape
 
 
+class broadcast(object):
+    def __init__(self, array1, array2):
+        self._a1 = array1
+        self._a2 = array2
+        #: The broadcast shape.
+        self.shape = broadcast.compute_broadcast_shape(array1.shape,
+                                                       array2.shape)
+
+    @staticmethod
+    def compute_broadcast_shape(shape1, shape2):
```

Comment:
Everything in this `broadcast` class *except* for this static `compute_broadcast_shape()` method appears to be unused. So perhaps just ditch the class and make this a function on a par with `_sliced_shape()` and `_full_keys()`?